jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -130,6 +130,14 @@ def parse_data(app, file_obj, data_type=None, data_label=None):
         raise NotImplementedError(f'Unsupported data format: {file_obj}')
 
 
+def _get_celestial_wcs(wcs):
+    """ If `wcs` has a celestial component return that, otherwise return None """
+    if hasattr(wcs, 'celestial'):
+        return wcs.celestial
+    else:
+        return None
```

Comment:
I think can be simplified but no biggie if you do not accept this.
```suggestion
    return wcs.celestial if hasattr(wcs, 'celestial') else None
```

Reply:
Still not sure how to do this if we never want true GWCS support in Cubeviz. Headache for another day!